pandas/tseries/tests/test_resample.py

```diff
@@ -660,6 +660,20 @@ def test_resample_empty(self):
         rs = xp.resample('A')
         assert_frame_equal(xp, rs)
 
+        # These were sometimes causing a segfault (for the functions with
+        # bounds-checking disabled) or an IndexError.  We just run them to
+        # ensure they no longer do.  (#10228)
+        index = pd.DatetimeIndex([])
```

Comment:
Cool, I ended up making these generators that returned the index instances, which I thought would keep the test cleaner.  Let me know if you prefer to just return a list of functions though and I can change it.